datapipelines/src/main/kotlin/io/customer/datapipelines/config/ScreenView.kt

```diff
@@ -0,0 +1,28 @@
+package io.customer.datapipelines.config
+
+/**
+ * Enum class to define how CustomerIO SDK should handle screen view events.
+ */
+enum class ScreenView {
```

Comment:
Curious why we are opting for making this an enum instead of:
- Boolean to allow screen analytics
- If in app is enabled, we always use screen views for in-app